code/datums/quirks/negative_quirks/all_nighter.dm

```diff
@@ -0,0 +1,38 @@
+/datum/quirk/all_nighter
+	name = "All Nighter"
+	desc = "You didn't get any sleep last night, and people can tell! You'll constantly be in a bad mood and will have a tendency to sleep longer."
+	icon = FA_ICON_BED
+	value = -4
+	mob_trait = list(TRAIT_ALL_NIGHTER, TRAIT_HEAVY_SLEEPER)
```

Comment:
`mob_trait` does not accept lists, you have to rewrite it to allow this

Reply:
... Actually you don't *need* `TRAIT_ALL_NIGHTER`, you don't use it anywhere, so you can stick to this just giving out `TRAIT_HEAVY_SLEEPER`.

Reply:
removed it, thanks

Reply:
actually, no, i still need to remove where i actually defined the trait

Reply:
ok now its gone